ServiceWebsite/ServiceWebsite/ClientApp/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "target": "es2015",
+    "target": "es5",
```

Comment:
is this automatically been updated or intentional?

Reply:
intentional